### ISSUE TITLE
Anisotropic Filtering not working in Halo Custom Edition

### ISSUE THREAD

**Issue by donizettilorenzo** [Kavawuvi/chimera#15]
Or so it seems! (using chimera_af variable)
Perfectly working on the same maps in Halo Combat Evolved (I use the lastest release of Vaporeon refined maps for Halo Combat Evolved).

**Comment by Kavawuvi** [comment]
Hello! I'm not able to reproduce this bug using the latest release with `chimera_af`. It appears to be working as intended for me.

That said, I don't tend to use it, instead opting for [dgVoodoo](http://dege.freeweb.hu/dgVoodoo2/dgVoodoo2.html)'s anisotropic filtering as it affects all bitmaps, not just ground textures.

**Comment by Vaporeon** [comment]
The `chimera_af` command simply enables Halo's own anisotropic filtering feature. If there is some condition where Halo's implementation does not work then `chimera_af` won't help you.
We have tested this on multiple GPU's and configurations, `chimera_af` works as intended and enables the feature within Halo.

What I think it happening is Custom Edition's "fast shaders" are possibly somehow causing the anisotropic filtering to not function on your GPU. The shaders for the game were changed in Custom Edition in an attempt to make them faster and many things were broken in the process.
